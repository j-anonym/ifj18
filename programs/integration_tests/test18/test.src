# Operations with int float and string
x= 10/4
print x,"\n"
x = 10.5/4
print x,"\n"
x = 95.55/4.95
print x,"\n"

a = 5
b = 1.3
x = a + b
print x,"\n"
x = a - b
print x,"\n"
x = a /b
print x,"\n"
x = a*b
print x,"\n"

c = "AA"
d = "BB"
x = c + d
print x,"\n"
