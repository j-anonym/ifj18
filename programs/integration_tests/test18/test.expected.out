2
0x1.5p+1
0x1.34d9364d9364dp+4
0x1.9333333333333p+2
0x1.d99999999999ap+1
0x1.ec4ec4ec4ec4ep+1
0x1.ap+2
AABB
